src/dataflow/src/render/sources.rs

```diff
@@ -67,19 +66,40 @@ pub(crate) fn import_table<G>(
     scope: &mut G,
     id: SourceInstanceId,
     persisted_name: Option<String>,
-) -> (
-    LocalInput,
-    (Collection<G, Row, Diff>, Collection<G, DataflowError, Diff>),
-)
+) -> (Collection<G, Row, Diff>, Collection<G, DataflowError, Diff>)
```

Comment:
Under discussion, but for posterity: if we continue to return the input, rather than have this method insert it, we can reduce the amount of `StorageState` that `import_table` needs to know about, and potentially also remove that pesky `inspect` (as the dangling dataflow will be in place at that point, due at least to the asof operator).